src/cbc_sdk/endpoint_standard/usb_device_control.py

```diff
@@ -629,6 +686,7 @@ class USBDeviceQuery(BaseQuery, QueryBuilderSupportMixin, CriteriaBuilderSupport
     """Represents a query that is used to locate USBDevice objects."""
     VALID_STATUSES = ["APPROVED", "UNAPPROVED"]
     VALID_FACET_FIELDS = ["vendor_name", "product_name", "endpoint.endpoint_name", "status"]
+    VALID_EXPORT_FORMATS = ('CSV', 'JSON')
```

Comment:
Formats are case insensitive in the backend so validation needs to be relaxed.

Reply:
I set it to use a case-smashed comparison to upper case.